Subject: mm/memcontrol: Disable on PREEMPT_RT
From: Thomas Gleixner <tglx@linutronix.de>
Date: Sun, 25 Jul 2021 21:35:46 +0200

559271146efc ("mm/memcg: optimize user context object stock access") is a
classic example of optimizing for the cpu local BKL serialization without a
clear protection scope.

Disable MEMCG on RT for now.

Signed-off-by: Thomas Gleixner <tglx@linutronix.de>
---
 init/Kconfig |    1 +
 1 file changed, 1 insertion(+)

--- a/init/Kconfig
+++ b/init/Kconfig
@@ -910,6 +910,7 @@ config PAGE_COUNTER
 
 config MEMCG
 	bool "Memory controller"
+	depends on !PREEMPT_RT
 	select PAGE_COUNTER
 	select EVENTFD
 	help
